cmd/containerd-shim/main_unix.go

```diff
@@ -83,23 +87,36 @@ func init() {
 	flag.Parse()
 }
 
-func main() {
+func setRuntime() {
 	debug.SetGCPercent(40)
 	go func() {
 		for range time.Tick(30 * time.Second) {
 			debug.FreeOSMemory()
 		}
 	}()
-
-	if debugFlag {
-		logrus.SetLevel(logrus.DebugLevel)
-	}
-
 	if os.Getenv("GOMAXPROCS") == "" {
 		// If GOMAXPROCS hasn't been set, we default to a value of 2 to reduce
 		// the number of Go stacks present in the shim.
 		runtime.GOMAXPROCS(2)
 	}
+}
+
+func main() {
+	parseFlags()
+	if versionFlag {
+		fmt.Printf("%s:\n", filepath.Base(os.Args[0]))
```

Comment:
Actually; this one is also only used for `containerd-shim`, so we can hard-code it here as well